curvefs/src/client/inode_cache_manager.cpp

```diff
@@ -44,12 +44,6 @@ CURVEFS_ERROR InodeCacheManagerImpl::GetInode(uint64_t inodeid,
         }
     }
 
-    curve::common::WriteLockGuard lg(mtx_);
-    bool ok = iCache_->Get(inodeid, &out);
-    if (ok) {
-        return CURVEFS_ERROR::OK;
-    }
-
     Inode inode;
     MetaStatusCode ret2 = metaClient_->GetInode(fsId_, inodeid, &inode);
```

Comment:
If not lock GetInode here， when an update inode and read inode concuently call， the read inode maybe return old version of inode